packages/shared/src/graphql/alerts.ts

```diff
@@ -12,6 +12,8 @@ export type Alerts = {
   lastBanner?: string;
   banner?: boolean;
   showStreakMilestone?: boolean;
+  lastBootPopup?: Date;
+  bootPopup?: boolean;
```

Comment:
Because API uses the same date structure.
If we rely on apps it can use a different date comparison to what's stored.

It's a common practice we follow for the other dates as well basically.